bench/benchmark.html

```diff
@@ -0,0 +1,53 @@
+<html>
+  <head>
+    <title>Glimmer Benchmark</title>
+    <script src='./benchmark.js'></script>
+    <script src="../assets/loader.js"></script>
+    <script>loader.noConflict({ define: 'enifed' });</script>
+    <script src="../amd/glimmer-common.amd.js"></script>
+    <script src="../amd/glimmer-compiler.amd.js"></script>
+    <script src="../amd/glimmer-runtime.amd.js"></script>
+    <script src="../amd/glimmer-benchmarks.amd.js"></script>
+
+    <style>
+      * {
+        box-sizing: border-box;
+      }
+
+      #cycle {
+        border: 1px solid #999;
+        padding: 5px;
+      }
+
+      #cycle:empty::content {
+        content: "Waiting to benchmark";
+      }
+
+      .histogram {
+        position: relative;
+        background: #efefef;
+        max-width: 1000px;
+        height: 200px;
+        margin-bottom: 20px;
+      }
+
+      .histogram .bar {
+        position: absolute;
+        background: linear-gradient(90deg, rgb(81,167,249) 0%,rgb(3,101,192) 100%);
+      }
+
+      .histogram .label {
+        position: absolute;
+        font-family: monospace;
+      }
+    </style>
+  </head>
+
+  <body>
+
+    <script>
+      require('glimmer-benchmarks').Benchmark.init();
+    </script>
+
+  </body>
+</html>
```

Comment:
Also believe this file should be dropped(at least until we have instrumentation). I am working on instrumenting Igniter with heimdall to provide benchmarking and perf tooling, I don't know yet if the output will work with this setup though.